README.md

```diff
@@ -23,12 +75,48 @@ It will listen on `0.0.0.0:8314`
 }
 ```
 
-## Geography database
+### 2. InfluxDB
+We currently only support InfluxDB version 1.8 and before
+
+### 3. The flag `--influxdb-continuous-query-period`
```

Comment:
Should we rename it to `--query-period`? It doesn't matter to the user how continuous query works.